src/components/CanvasSaveButton/CanvasSaveButton.js

```diff
@@ -0,0 +1,150 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import OLMap from 'ol/Map';
+import Button from '../Button';
+
+const propTypes = {
+  /**
+   * Title of the button.
+   */
+  title: PropTypes.string.isRequired,
+
+  /**
+   * Children content of the button.
+   */
+  children: PropTypes.element.isRequired,
+
+  /**
+   * CSS class of the button.
+   */
+  className: PropTypes.string,
+
+  /**
+   * Translation function.
+   */
+  t: PropTypes.func,
```

Comment:
I don't think we need the t function for the CanvasSaveButton since we don't need it for Button.